transforms/universal/resize/spark/Dockerfile

```diff
@@ -0,0 +1,44 @@
+ARG BASE_IMAGE=quay.io/dataprep1/data-prep-kit/data-prep-kit-spark-3.5.2:0.2.1.dev0
```

Comment:
0.2.1.dev0 is no longer used in dev.  to be consistent with other spark transforms (recent change), use `latest` as the tag.  Note however, that this is generally overridden from the Makefile anyway by setting BASE_IMAGE when docker build is called.  But for consistency it would be nice to change to `latest`.